app_dart/lib/src/request_handlers/update_task_status.dart

```diff
@@ -144,28 +111,9 @@ class UpdateTaskStatus extends ApiRequestHandler<UpdateTaskStatusResponse> {
 
   /// Retrieve [Task] to update from [DatastoreService].
   Future<Task> _getTask(DatastoreService datastore) async {
```

Comment:
_getTask should be removed, and directly call _getTaskFromNamedParams

Reply:
done